### DISCUSSION TITLE
Useless "Welcome to GitHub Copilot" permanent sidebar panel in IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by lincentpega** [community/community#72934] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Please, remove this permanent "Welcome to GitHub Copilot" sidebar panel. It just makes a mess in the interface and produces zero value containing just two welcome messages
<img width="298" alt="image" src="https://github.com/community/community/assets/99477948/ea44443e-5564-4fa9-91da-1afd3a245201">

**Comment by mousedownmike** [comment]
At the very least, have the ability to disable the panel in settings so those settings can be sync'd across all IDEs.  I've had to "Hide" this panel 12 times in the last few days between IDEA, Goland, and Pycharm.

**Comment by msangel** [comment]
Any updates on this? Can I completely remove this panel from my IDEA once I have passed the tour?

**Reply by radiobone** [reply]
I just right clicked it then clicked "Hide" to remove it.

**Reply by lincentpega** [reply]
It's still unable to remove this stuff completely, this icon is still present in the hidden list, while it's still useless

**Reply by radiobone** [reply]
I keep my IDE open 24/7 so I wouldn't know 😝

**Reply by lincentpega** [reply]
It doesn't appears once it's hidden, its just still present in panel here. Imagine every extension developer thinking his windows are so important so you'll be unable to remove them and stack in this hidden menu xD
<img width="254" alt="image" src="https://github.com/community/community/assets/99477948/751bc134-362d-4557-a0db-f1eef1059f1e">

**Reply by radiobone** [reply]
I never noticed that... now I'll notice it everytime!

**Comment by Ry-DS** [comment]
Also voicing my frustration for this panel. I don't understand why this was thought to be a good idea; a global turn off would be much appreciated.

**Comment by imwule** [comment]
That's a f**king shit feature.

**Comment by edwinramirez** [comment]
Adding a comment to keep this alive. Please add a way to remove this panel

**Comment by maluramichael** [comment]
Yeah you can hide the panel but it pops up in every new project. The setting is not persistent across multiple projects. Hate it :/

**Reply by joseluisgs** [reply]
Please remove it!!!

**Comment by Seayon** [comment]
Still can't remove it ,f**king shit feature.

**Comment by jxerome** [comment]
Please remove this useless panel.

**Comment by tksrc** [comment]
+1 please remove this. One time pop up is enough.

**Comment by frgreiner** [comment]
I am also missing a global `[] Don't show "Welcome to GitHub Copilot" again` setting in Intellij

Edit: Seems like i have to hide it once per project/repository (~50+ times)

**Comment by haslingerm** [comment]
How is there still not even an answer from the devs? 
I open dozens of projects each day (student submissions) and have to hide it every single time, because it annoys me like crazy.

**Comment by remi-espie** [comment]
Please devs 🙏

**Comment by gitHubhuanhuan** [comment]
hi ，Can I borrow your copilot? I can't use my student bag, thank you，Maybe you can activate it for me，[sit back and wait](javascript:;)

**Comment by adamkubon** [comment]
I vote to remove this "anti-feature"

**Comment by SepehrNazari** [comment]
Please remove this tab, it's EXTREMELY annoying to me and I won't pay for copilot unless this is removed

**Comment by cmcconomyfwig** [comment]
Outrageous - get rid of this thing!

**Comment by PerfectlyFineCode** [comment]
Been a year with no status updates, kind of disappointing.

**Comment by br45rm** [comment]
Wtf, is this not removable???????? Who thought it was a good idea to add this cluttering garbage to my workspace?

**Comment by seanzxx** [comment]
I really hate this useless welcome panel. 
Could you please remove it completely?

**Comment by TimmJensen** [comment]
+1

**Comment by tindvik** [comment]
I would suggest looking in the `~/.config/JetBrains/IntelliJIdea2024.3/options` folder, and find the github-copilot configuration. 
There might be a boolean there that could toggle it off.

Disclaimer, I don\t use CoPilot, so I have not done this myself

**Comment by tresende** [comment]
any updates?

**Comment by Zaubentrucker** [comment]
I'd really apprechiate if this icon could be disabled, as it adds itself every time that I open a new project
